src/components/ProjectConfigurationModal/ProjectConfigurationModal.js

```diff
@@ -132,15 +128,23 @@ class ProjectConfigurationModal extends Component<Props, State> {
                 ))}
               </ProjectIconWrapper>
             </FormField>
-            <Button
-              icon={<IconBase icon={edit2} />}
-              onClick={this.saveSettings}
-              disabled={!queueEmpty}
-            >
-              Save
-            </Button>
-            {!queueEmpty && 'Waiting for pending tasks to finish.'}
-          </FadeIn>
+
+            <Actions>
+              <FillButton
+                size="large"
+                colors={[COLORS.green[700], COLORS.lightGreen[500]]}
+                disabled={!dependenciesChangingForProject}
+              >
+                Save Project
+              </FillButton>
```

Comment:
I removed the icon because we haven't typically used icons in buttons for actions (the only ButtonWithIcon being used is for toggling between Vanilla React and Gatsby projects).

I'm not against having an icon in this button, but it does mean we'd need to refactor ButtonWithIcon, which is a bigger job than I wanted to take on.

Reply:
Note: This button will do the form submit but it may require the `onClick` prop. If you're finding an alternative way to handle the submit on enter key inside project name input you can use this to submit but if you're closing the form earlier you need to have the click handler.